test/fail_compilation/nortti_mini.d

```diff
@@ -0,0 +1,6 @@
+/**
+ * EXTRA_SOURCES: imports/nortti.d
+ * REQUIRED_ARGS: -betterC -Irunnable/extra-files/miniRT -defaultlib= -release
+ */
+
+// Real test is in imports/nortti.d
```

Comment:
If you don't test for the error output, it's unclear whether everything is checked.